livekit_internal.proto

```diff
@@ -63,6 +63,11 @@ message NodeStats {
   float load_avg_last5min = 19;
   float load_avg_last15min = 20;
   float cpu_load = 21;
+  uint32 sys_packets_out = 28;
```

Comment:
how does this differ from `packets_out` ?

Reply:
I think it will include everything at network stack/interface level.